src/app/features/genes/components/gene-nominated-targets/gene-nominated-targets.component.ts

```diff
@@ -120,37 +120,10 @@ export class GeneNominatedTargetsComponent implements OnInit {
         inputDataArray = this.commaFlattenArray(inputDataArray);
 
         // Populate targetNomination display fields
-        de.teams_display_value = '';
-        if (teamsArray.length) {
-          de.teams_display_value = teamsArray
-            .filter(this.getUnique)
-            .sort((a: string, b: string) => a.localeCompare(b))
-            .join(', ');
-        }
-
-        de.study_display_value = '';
-        if (teamsArray.length) {
```

Comment:
Also fixes this case, where `studyArray` was conditionally formatted based on the `teamsArray.length`, rather than the `studyArray`

Reply:
nice